src/steps/10-FinancialDetails/MIPR.vue

```diff
@@ -88,24 +88,26 @@ export default class MIPR extends Mixins(SaveOnLeave)  {
     try {
       this.saved = await FinancialDetails.loadFundingRequestMIPRForm();
       this.MIPRNumber = this.saved.mipr_number;
-      const attachment = await Attachments.getAttachmentById({
-        serviceKey: FUNDING_REQUEST_MIPRFORM_TABLE, sysID: this.saved.mipr_attachment});
-      if (attachment) {
-        const file = new File([], attachment.file_name, {
-          lastModified: Date.parse(attachment.sys_created_on || "")
-        });
-        const upload: uploadingFile = {
-          attachmentId: attachment.sys_id || "",
-          fileName: attachment.file_name,
-          file: file,
-          created: file.lastModified,
-          progressStatus: 100,
-          link: attachment.download_link || "",
-          recordId: attachment.table_sys_id,
-          isErrored: false,
-          isUploaded: true
+      if(this.saved.mipr_attachment){
```

Comment:
You could have gone either way on this "if" condition, where this could have been handled inside the Attachments. getAttachmentById. But this implementation also works. Approving this but just double check if the call to getAttachmentById is used elsewhere by other components. And if it call by multiple components, then the fix could be moved inside getAttachmentById.